app/graphql/types/subscriptions/create_subscription_input.rb

```diff
@@ -14,6 +14,7 @@ class CreateSubscriptionInput < BaseInputObject
 
       argument :billing_time, Types::Subscriptions::BillingTimeEnum, required: true
       argument :subscription_at, GraphQL::Types::ISO8601DateTime, required: false
+      argument :ending_at, GraphQL::Types::ISO8601DateTime, required: false
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `ending_at` should appear before `subscription_at`.